src/app/_api/explore/getNotificationAllChecked.ts

```diff
@@ -0,0 +1,9 @@
+import axiosInstance from '@/lib/axios/axiosInstance';
+
+const getNotificationAllChecked = async () => {
+  const response = await axiosInstance.get(`/alarms/check-new`);
```

Comment:
```suggestion
  const interface alarmCheckType {
       isAllChecked: boolean
  }
  //...
  const response = await axiosInstance.get<boolean>(`/alarms/check-new`);
```

너무 간단한 api라 불필요할 것 같기도 하지만,
혹여나 의도적으로 빼신 게 아닐까봐 언급합니다!!

Reply:
오매낫 나중에 써야지 하고 완전 까먹었는데 감사합니다..!!😆